test/functional/artefacts_controller_test.rb

```diff
@@ -152,6 +152,24 @@ class ArtefactsControllerTest < ActionController::TestCase
         get :new
         assert_select "li[class~=active] a[href=/artefacts/new]"
       end
+
+      context 'if publisher is an app for which the taging has NOT been migrated' do
+
```

Comment:
Can we keep the usage of whitespace separators uniform? It helps to pick out specs and contexts when quickly scanning through the code.